WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -105,7 +105,11 @@ public class BlockEditorScreen: BaseScreen {
     }
 
     public func publish() throws -> EditorNoticeComponent {
-        publishButton.tap()
+        var tries = 0
+        repeat {
+            publishButton.tap()
+            tries += 1
+        } while !publishNowButton.waitForIsHittable(timeout: 3) && tries <= 3
```

Comment:
I'm not sure about this code. Have you been able to see it in action?

I think if the element on which `tap()` is called can't be found, the test would fail, going out of the `repeat` loop.

Reply:
Yeah, I tested a few cases.

You're right, that's what would happen. The `publishButton` will always be there in the editor screen, but it will be disabled until we start typing into the editor screen and the `tap()` doesn't break if the button is disabled.

Maybe adding a `guard publishButton.waitForIsHittable() else { return }` before entering the `repeat` loop would make the test fail faster in case something went wrong on the previous steps.

Reply:
I'll make the changes already [in the new branch](https://github.com/wordpress-mobile/WordPress-iOS/tree/bloggig-reminders-dismissal-mokagio-edits).

Reply:
> UPDATE: I think we are covered here because...

Ah, right. I had overlooked the fact that we are working with two different elements `publishButton` and `publishNowButton`. 👍 

I still hope that it won't be necessary to add this. I'll try to rebase the `ScreenObject` branch now that the UI tests run in the Xcode 13 CI and see how we go.

In case the flakiness is not addressed, I have a suggestion: could you add a few comments to document _why_ we are doing this check?

Reply:
I did add the comments but I hope we don't need them as we move forward with the `ScreenObject`.